org.eclipse.xtend.lib.gwt.test/.classpath

```diff
@@ -7,7 +7,12 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
+	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.web.container"/>
```

Comment:
I doubt that this entry is good. It would require JST to be installed, which is often not the case.

Reply:
id rather go with it if that gets us rid of the errors